backend/open_webui/utils/webhook.py

```
@@ -80,9 +79,10 @@ def post_webhook(name: str, url: str, message: str, event_data: dict) -> bool:
             payload = {**event_data}
 
         log.debug(f"payload: {payload}")
-        r = requests.post(url, json=payload)
-        r.raise_for_status()
-        log.debug(f"r.text: {r.text}")
+        async with aiohttp.ClientSession() as session:
+            async with session.post(url, json=payload) as response:
+                response.raise_for_status()
+                await response.text()
         return True
```

Ruff (0.13.1)

Consider moving this statement to an `else` block

(TRY300)